src/modules/scoreboardUI.js

```diff
@@ -0,0 +1,15 @@
+const scoreListElement = document.querySelector('#score-list');
+
+export function addScoreToUI(data) {
+  const listElement = document.createElement('li');
+  listElement.innerText = `${data.user}: ${data.score}`;
+  scoreListElement.appendChild(listElement);
+}
+export function showScoresToUI(list) {
+  scoreListElement.innerHTML = '';
+  list.forEach((element) => {
+    const listElement = document.createElement('li');
+    listElement.innerText = `${element.user}: ${element.score}`;
+    scoreListElement.appendChild(listElement);
+  });
+}
```

Comment:
- [ ] It is required to use arrow functions for all your functions instead of function keyword. Kindly change this to arrow function and replicate it to every other concerned functions. Below is the section of the project requirements that stated this

![requirement-point](https://user-images.githubusercontent.com/26597432/195989320-0b30c93e-92c3-4946-940d-c11487ab523f.png)